app.py

```diff
@@ -133,11 +133,11 @@ def predict(uuid):
 		query = pd.DataFrame([json_])
 		return jsonify({'prediction': list(model.predict(query))})
 
-	except Exception, e:
-		print 'error'
-		print e
+	except getopt.GetoptError as e:
```

Comment:
In python3 the raise syntax no longer accepts comma-separated arguments.

Use as instead:

except getopt.GetoptError as e:
This form is also backwards-compatible with 2.6 and 2.7.